showmore_dyn/showmore_dyn.js

```diff
@@ -0,0 +1,81 @@
+$(document).ready(function(){
+        handleNewWallItemBodies();
+
+        var mutationObserver = new MutationObserver(function(mutations) {
+                handleNewWallItemBodies();
```

Comment:
I assume this is meant to add the show more to dynamically added new items? We actually have a Javascript hook for this that would trigger possibly less often than this: https://friendica.mrpetovan.com/help/Addons#JavaScript+hooks

Reply:
Ok, will look into it.